pkg/webhook/vlanconfig/validator.go

```diff
@@ -23,9 +23,11 @@ import (
 )
 
 const (
-	createErr = "could not create vlanConfig %s because %w"
-	updateErr = "could not update vlanConfig %s because %w"
-	deleteErr = "could not delete vlanConfig %s because %w"
+	createErr                           = "could not create vlanConfig %s because %w"
+	updateErr                           = "could not update vlanConfig %s because %w"
+	deleteErr                           = "could not delete vlanConfig %s because %w"
+	StorageNetworkNetAttachDefNamespace = "harvester-system"
+	StorageNetworkAnnotation            = "storage-network.settings.harvesterhci.io"
```

Comment:
`storage-network` should also define those to consts for common usage, let's update Harvester consts in another PR ,  later change here

```
pkg/controller/master/storagenetwork/storage_network.go:	StorageNetworkAnnotation        = "storage-network.settings.harvesterhci.io"
pkg/controller/master/storagenetwork/storage_network.go:	longhornStorageNetworkName = "storage-network"
```

Reply:
yes, they are not in common place now